SPECS/keepalived/keepalived.spec

```diff
@@ -74,22 +92,26 @@ fi
 %systemd_postun_with_restart keepalived.service
 
 %files
-%defattr(-,root,root)
-%license COPYING
-%doc %{_docdir}/%{name}/README
-%{_sbindir}/%{name}
-%{_bindir}/genhash
-%{_unitdir}/%{name}.service
-%config(noreplace) %{_sysconfdir}/sysconfig/%{name}
-%config(noreplace) %{_sysconfdir}/%{name}/%{name}.conf
+%defattr(-, root, root, -)
+%doc AUTHOR ChangeLog CONTRIBUTORS COPYING README TODO
+%doc doc_install/*
+%dir %{_sysconfdir}/keepalived/
+%attr(0600,root,root) %config(noreplace) %{_sysconfdir}/keepalived/keepalived.conf
+%attr(0600,root,root) %config(noreplace) %{_sysconfdir}/sysconfig/keepalived
+%{_unitdir}/keepalived.service
 %{_datadir}/snmp/mibs/KEEPALIVED-MIB.txt
 %{_datadir}/snmp/mibs/VRRP-MIB.txt
 %{_datadir}/snmp/mibs/VRRPv3-MIB.txt
+%{_bindir}/genhash
+%attr(0755,root,root) %{_sbindir}/keepalived
 %{_mandir}/man1/genhash.1*
-%{_mandir}/man5/%{name}.conf.5*
-%{_mandir}/man8/%{name}.8*
+%{_mandir}/man5/keepalived.conf.5*
+%{_mandir}/man8/keepalived.8*
 
 %changelog
+* Tue Feb 08 2022 Cameron Baird <cameronbaird@microsoft.com> - 2.2.7-1
+- Update source to v2.2.7
```

Comment:
This change looks like we've used a different distro's spec. Could you mention the distro in the changelog along with the license under which they allow others to use it. Example for Fedora: `Using Fedora [version] spec (license: MIT) for guidance.`